### DISCUSSION TITLE
How to Hide series when Chart Legend click

### DISCUSSION THREAD

**Discussion by Ashok4a9** [Live-Charts/LiveCharts2#1475] [Q&A]
We have created line chart its rendering is fine. in top of the chart we are showing legends. my requirement is when i click the legend i want to hide corresponding line series. how can i achieve this.

**Comment by Xixi0506** [comment]
You can refer to this section of the [Legends](https://livecharts.dev/docs/WPF/2.0.0-rc2/CartesianChart.Legends)

There is a PointerDown event which you should handler it.
```
private static VisualElementHandler<SkiaSharpDrawingContext> GetPointerDownHandler(
    IChartSeries<SkiaSharpDrawingContext> series)
{
    return (visual, args) => { series.IsVisible = !series.IsVisible; };
}
```
But I think this feature should be set as a default option.
